projects/web_agency_hero/index.html

```diff
@@ -1,10 +1,17 @@
 <!DOCTYPE html>
 
-<html>
+<html lang="en">
   <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Web Agency Hero</title>
     <link rel="stylesheet" href="styles.css" />
   </head>
-  <body></body>
+  <body>
+    <h1>Dave's Devs <br />& Designers</h1>
+    <p>We're a group devs & <br />designers who will help your <br />dream <span class="underline">come true</span>.</p>
+    <button>Let's chat</button>
```

Comment:
Note:
1. These text elements are centred vertically and have a left margin.
2. You seem not to have imported the [Open Sans](https://fonts.google.com/specimen/Open+Sans) font.

Please, have a look at the design.